internal/unix/types_linux.go

```diff
@@ -30,6 +30,7 @@ const (
 	PerfBitWatermark         = linux.PerfBitWatermark
 	PERF_SAMPLE_RAW          = linux.PERF_SAMPLE_RAW
 	PERF_FLAG_FD_CLOEXEC     = linux.PERF_FLAG_FD_CLOEXEC
+	RLIM_INFINITY            = linux.RLIM_INFINITY
```

Comment:
I think it makes sense to swap the whole codebase over to `unix.RLIM_INFINITY` to get rid of `math` dependency, but I didn't want to do it in this PR since it's kinda unrelated. I can follow up on that later.

Reply:
Yeah, that's fine